apis/database/v1beta1/rdsinstance_types.go

```diff
@@ -109,10 +109,39 @@ type ScalingConfiguration struct {
 // S3RestoreBackupConfiguration defines the details of the S3 backup to restore from.
 type S3RestoreBackupConfiguration struct {
 	// BucketName is the name of the S3 bucket containing the backup to restore.
+	// +optional
+	// +crossplane:generate:reference:type=github.com/crossplane/provider-aws/apis/s3/v1beta1.Bucket
 	BucketName *string `json:"bucketName"`
```

Comment:
```suggestion
	BucketName *string `json:"bucketName,omitempty"`
```
Since this is optional now. I'm assuming the referencer system also handles raising an error if all 3 of the `BucketName` fields are not set?